easybuild/framework/easyconfig/parser.py

```diff
@@ -209,7 +214,7 @@ def get_config_dict(self, validate=True):
             self._formatter.validate()
 
         cfg = self._formatter.get_config_dict()
-        self.check_values_types(cfg)
+        self.check_values_types(cfg, self.auto_convert)
```

Comment:
no need to pass this down, just use `self.auto_convert` in `self.check_values_types`?